src/rosdistro/manifest_provider/git.py

```diff
@@ -87,9 +85,9 @@ def _temp_git_clone(url, ref):
     base = tempfile.mkdtemp('rosdistro')
     git = Git(cwd=base)
     try:
-        if git.version_gte('1.8.0') and not _ref_is_hash(ref):
-            # Directly clone the required ref with least amount of additional history.
-            # Available since git 1.8.0, but only works for tags and branches, not hashes:
+        if git.version_gte('1.8.0') and not ref_is_hash(ref):
+            # Directly clone the required ref with least amount of additional history. This behaviour
+            # has been available since git 1.8.0, but only works for tags and branches, not hashes:
```

Comment:
Can you please undo the comment change? I think this was something we iterated on in the other PRs and was intentionally changed to the shorter left side.